### PR TITLE
DDF-3297 disable security manager when running using the service wrapper

### PR DESCRIPTION
**Pull request by paouelle**
#### What does this PR do?
Disables the security manager when running using the service wrapper.
#### Who is reviewing it? 
@figliold @tbatie  
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@clockard
@lessarderic
@stustison
@coyotesqrl
#### What are the relevant tickets?
[DDF-3297](https://codice.atlassian.net/browse/DDF-3297)
[DDF-3273](https://codice.atlassian.net/browse/DDF-3273)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by paouelle** [comment]
build now

**Comment by paouelle** [comment]
build now

File: distribution/ddf-common/src/main/resources/bin/setenv-windows-wrapper.conf [hunk 1]
```diff
@@ -33,12 +33,13 @@ wrapper.java.additional.23 = -Dorg.tanukisoftware.wrapper.WrapperManager.mbean=T
 #wrapper.java.additional.24 = -XX:OnOutOfMemoryError=%KARAF_HOME%/bin/ddf_on_error.bat wrapper %p
 #wrapper.java.additional.25 = -XX:OnError=%KARAF_HOME%/bin/ddf_on_error.bat wrapper %p
 
+# The Security Manager is turned off by default in DDF.
```
Comment (lessarderic) [inline distribution/ddf-common/src/main/resources/bin/setenv-windows-wrapper.conf]: ❓ Should we add a note like we did on L32, i.e., uncomment and re-number?

File: distribution/ddf-common/src/main/resources/bin/setenv-wrapper.conf [hunk 2]
```diff
@@ -35,13 +35,13 @@ wrapper.java.additional.24 = -Djava.security.egd=file:/dev/./urandom
 wrapper.java.additional.25 = -XX:OnOutOfMemoryError=%KARAF_HOME%/bin/ddf_on_error.sh wrapper %p
 wrapper.java.additional.26 = -XX:OnError=%KARAF_HOME%/bin/ddf_on_error.sh wrapper %p
 
+# The Security Manager is turned off by default in DDF.
```
Comment (lessarderic) [inline distribution/ddf-common/src/main/resources/bin/setenv-wrapper.conf]: ❓ Same question as above.

**Comment by lessarderic** [comment]
@paouelle, does this need to be back-ported to 2.11.x?

**Comment by paouelle** [comment]
I have the same changes done in the ticket for the OOM/error script support on 2.11.x. This one is because the OOM/error script support has been merged already on master.

**Comment by paouelle** [comment]
build now

**Comment by paouelle** [comment]
build now

**Comment by jhunzik** [comment]
Successfully heroed, installed DDF with a service wrapper and verified the security manager was not installed.

**Pull request merged**